src/pages/index.js

```diff
@@ -30,8 +111,17 @@ function fillProfileForm(e) {
 const profileEditForm = new PopupWithForm(
   "#profile-edit-popup",
   (inputValues) => {
-    pageUserInfo.setUserInfo(inputValues);
-    profileEditForm.close();
+    profileEditForm.setLoadingState(true);
+    api.updateUserInfo(inputValues).then(() => {
+      api
```

Comment:
Needs correction: There is no need to get the user here, you should update the info using `updateUserInfo` response

Reply:
corrected